### PR TITLE
adding info on alphanumeric characters and periods

### PR DESCRIPTION
**Pull request by mscharlock**
Updating to reflect current info on namespace and function id. They now can only contain alphanumeric characters and/or periods.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810112315072398-461%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GwQEOaRE56r3GeCBtX82x1iYkgMTDa4eZgcN2wPqI9M%3D&st=2018-10-11T23%3A11%3A45Z&se=2018-11-11T23%3A16%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810112356427142-461%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=PoswzeQpxjiPMgqRwOjrOo%2FCUKyYKLzYaxPkiHoM1xI%3D&st=2018-10-11T23%3A53%3A03Z&se=2018-11-11T23%3A58%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810112356529556-461%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=97NOFxow86J6RfPEwueI0rxmU6wZdyrtoub7iyYLKkE%3D&st=2018-10-11T23%3A54%3A31Z&se=2018-11-11T23%3A59%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810120000490845-461%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ro7iwmMoYCV3moo4Te0ysSxmIclipwUTcdT%2Bd9L13Tk%3D&st=2018-10-11T23%3A57%3A06Z&se=2018-11-12T00%3A02%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810120001555665-461%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dRC3DQIJOjL%2F9HPELGwZZA%2FbEaVb5uH4s9TAEQB9kAo%3D&st=2018-10-11T23%3A58%3A35Z&se=2018-11-12T00%3A03%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-best-practices.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ Keep in mind the following best practices when creating custom functions in your
 
 * In the JSON metadata file, specify the value of each `id` property in uppercase. Doing so makes it obvious which part of the `CustomFunctionMappings` statement in your JavaScript code corresponds to the `id` property in the JSON metadata file (provided that your function name uses camelCase, as recommended earlier).
 
+* It is required that the value of each `id` property contain alphanumeric characters. `id` property values *may* also contain periods.
```
Comment (kbrandl) [inline docs/excel/custom-functions-best-practices.md]: suggest rephrasing to:

In the JSON metadata file, ensure that the value of each `id` property contains only alphanumeric characters and periods.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810121733556731-461%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mIUfBLYJOgujI43acXExdlVzAKdn5uEAVMeLuvJf2O0%3D&st=2018-10-12T17%3A30%3A19Z&se=2018-11-12T17%3A35%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfidfix/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810121735288655-461%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Ax2A6Lh1iTZ6qUYJDycrR2mN6RtD5tt2wAakmEwrDLQ%3D&st=2018-10-12T17%3A31%3A48Z&se=2018-11-12T17%3A36%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**